java/src/com/liferay/nativity/control/win/WindowsNativityUtil.java

```diff
@@ -65,15 +65,17 @@ private static void _load() {
 
 			_loaded = true;
 
+		    System.out.println("Successfully loaded " + nativityDllName);
```

Comment:
I REALLY wanted feedback on whether the DLL was loaded or not, as it can save hours of pointless debugging. We could 

- refactor this function/base class/interface to have a boolean return type that we check in our app
- make this use our logger
- auto-redirect stdout in our logger (idk how possible this is)

Once our shell extensions reliably work, we can take them off.